client/src/routes.js

```diff
@@ -48,21 +48,21 @@ const router = new VueRouter({
     },
     {
       name: "auth",
-      path: "/login",
+      path: "/auth",
       component: UserAuth,
       meta: { requiresUnauth: true },
     },
     { name: "notFound", path: "/:notFound(.*)*", component: NotFound },
   ],
 });
 router.beforeEach((to, from, next) => {
-  if (to.path !== "/login/" && to.meta.requiredAuth) {

```

Comment:
samo misel.. jaz bi rajš primerjal med `to.name` in ne z pathom direkt ker se lahko spremeni..

Reply:
Bom spremenil